tests/licensedcode/data/license_db/license_dump/index.html

```diff
@@ -371,6 +371,7 @@
     </p>
     <p>Designed and built by <a href="https://www.nexb.com/" target="_blank">nexB</a>. Licensed under the <a href="cc-by-4.0.html">Creative Commons Attribution License 4.0 (CC-BY-4.0)</a>.</p>
     <p>Generated with <a href="https://github.com/nexB/scancode-toolkit" target="_blank">ScanCode toolkit</a> 32.0.0b1 on Dec 22, 2022.</p>
+    <p>This is updated daily automatically with latest updates from the develop branch of scancode-toolkit, if any.</p>
```

Comment:
we should IMHO tell which exact commit and link to it.
The `toolkit</a> 32.0.0b1 on Dec 22, 2022.</p>` thing is  misleading.